QueenCam/QueenCam/Presentation/Views/Guiding/Pen/PenWriteView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  PenView.swift
+//  QueenCam
+//
+//  Created by Bora Yun on 10/15/25.
+/** 펜 가이드라인 작성 뷰 (only 모델) */
+
+import SwiftUI
+
+struct PenWriteView: View {
+  var penViewModel = PenViewModel()
+  @State private var tempPoints: [CGPoint] = []  // 현재 그리고 있는 선의 좌표(임시)
+  var body: some View {
+    VStack {
+      GeometryReader { geo in
+        // MARK: - 실제 드로잉 영역
+        Canvas { context, size in
+          // 이미 strokes에 저장된 모든 선들 그리기
+          for stroke in penViewModel.strokes {
+            guard stroke.points.count > 1 else { continue }
+            var path = Path()  // Line을 담는 객체
+            path.addLines(stroke.points)
+            context.stroke(path, with: .color(.white), style: StrokeStyle(lineWidth: 8, lineCap: .round, lineJoin: .round))
+            context.stroke(path, with: .color(.orange), style: StrokeStyle(lineWidth: 4, lineCap: .round, lineJoin: .round))
```

Comment:
```suggestion
            context.stroke(path, with: .color(outerColor), style: StrokeStyle(lineWidth: 8, lineCap: .round, lineJoin: .round))
            context.stroke(path, with: .color(innerColor), style: StrokeStyle(lineWidth: 4, lineCap: .round, lineJoin: .round))
```

아직 하이파이 적용 전이지만 컬러는 프로퍼티로 따로 잡아주시면 유지보수가 더 쉬울 것 같다는 예감이 듭니당 (그냥 제안입니다!)